patroni/postgresql.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, config):
         self.replication_slots = []  # list of already existing replication slots
         self.retry = Retry(max_tries=-1, deadline=5, max_delay=1, retry_exceptions=PostgresConnectionException)
 
+        self._creating_replica = False
```

Comment:
And it would be better to return this status via the API as well instead of sysid == 0 nonsense.